panel/src/components/Sections/FieldsSection.vue

```diff
@@ -36,6 +36,11 @@ export default {
       return this.$store.getters["content/values"]();
     }
   },
+  watch: {
+    timestamp() {
```

Comment:
Since I was quite irritated where this is coming from... maybe we should in general add a comment what `timestamp` is/does?

Reply:
There's one huge benefit of the Fiber setup that we didn't fully use yet. Sections don't really need to watch for certain model changes anymore. I.e. when dialogs close. In such a case we always have a reloaded view. The easiest way to tell the sections about a reloaded view is actually such an updated timestamp that's coming from the backend with every view response. So basically any component can listen for those timestamp updates and then knows that the route/model has been updated. This will even be easier when we refactor sections and send them directly on view load, but for now this is a really great way to keep everything in sync.

Reply:
✅